### DISCUSSION TITLE
Question about Browser Usage

### DISCUSSION THREAD

**Discussion by guguxh** [wxyz-abcd/node-haxball#31] [Q&A]
![image](https://user-images.githubusercontent.com/61206153/205935320-41e70ea7-5251-46df-a5ff-dfccc2ea6d72.png)
![image](https://user-images.githubusercontent.com/61206153/205935358-4963f524-e2eb-4f2a-b982-82cce0e68778.png)
When I setup the overrides, and turn on, the game doesn't work properly, like in screenshots.

**Comment by wxyz-abcd** [comment]
I'm moving this to discussions, since this is not an issue but a question.

**Comment by wxyz-abcd** [comment]
You do not need to override anything. You need a little web development skills to convert this code into a working code. Otherwise, you can use the new https://abc-haxball-proxy.up.railway.app/ website for headless bot development purposes. Note that it is not always active for now because i'm using a free hosting, and you can not use it while it is not active. I'm planning to make it active from 10th of each month until the end of that month. If we somehow get financial support for the project, it could be up forever. Until then, we have to continue like this. (And even if you converted this code into a working code, it would still not work while the server is down.) You can however always use the npm package, it's independent of this server.

**Comment by wxyz-abcd** [comment]
I just updated the polyfill links in README.MD file. You can set them like I did, but unfortunately browser code depends on my server, therefore you will have to wait until the server is running.

You have to create a local html file and write the contents in it. Then you can open it directly in your browser, and it will work.
